enhancement/anti_ai_checker.py

```
@@ -0,0 +1,174 @@
+            elif issue_type == "monotonous_start":
+                score -= 5
+        
+        return max(0, score)
+    
+    def format_report(self, text: str) -> str:
+        """??????"""
+        issues = self.check_text(text)
+        score = self.get_score(text)
+        
+        lines = [
+            f"?AI?????",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)